### PR TITLE
Discussion - part 1

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

Add discussion

### Issue

#207 

#### Which areas should receive a particularly close look?
discussion

#### Is there anything that you want to discuss further?
no - majority of discussion here - will add more later

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by jharenza** [comment]
@sjspielman - I have a good bit of this written and in good shape, so if you want a break from analysis and want to review, we can get this chunk in so it doesn't become super long for review. In the meantime, I will work on Figs 4/5 text!

File: content/04.discussion.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
 ## Discussion
 
-Stub in discussion section   
+Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians across the nation to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
```
Comment (sjspielman) [inline content/04.discussion.md:3]: "across the nation" 

Do we have international collaborators?

File: content/04.discussion.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+To our knowledge, this was the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
```
Comment (sjspielman) [inline content/04.discussion.md:4]:
```suggestion
To our knowledge, this initiative represents the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
```

File: content/04.discussion.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and create over 40 code- and scientifically-reviewed scalable modules for common downstream cancer genomics analyses which have been used to create new knowledge as well as define new questions.
```
Comment (sjspielman) [inline content/04.discussion.md:5]:
```suggestion
We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and establish over 40 scalable modules that have undergone scientific and analytical code review and perform common downstream cancer genomics analyses.
```

Rephrasing suggestions based on wording in Figure 1 workflow panel. Also note sure that the phrase " which have been used to create new knowledge as well as define new questions" adds anything beyond what's already here

File: content/04.discussion.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+We detect and show expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
```
Comment (sjspielman) [inline content/04.discussion.md:6]:
```suggestion
We detected and showed expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
```

File: content/04.discussion.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+Molecular subtyping information was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
+Furthermore, this tumor cohort has yet to be profiled by DNA methylation.
```
Comment (sjspielman) [inline content/04.discussion.md:8]: Seems out of place. Does this mean the lack of methylation profiling is related to lack of subtyping? If so, rephrase to something like - "Furthermore, this tumor cohort has yet to be profiled by DNA methylation, increasing the challenge of determining molecular subtypes."

File: content/04.discussion.md (hunk 1 above)
Comment (jharenza) [reply content/04.discussion.md:3]: oh good point - we do have clinicians outside - was thinking of those who worked on GitHub specifically, but I can update!

File: content/04.discussion.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+We detect and show expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
+Molecular subtyping information was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
+Furthermore, this tumor cohort has yet to be profiled by DNA methylation. 
+Thus, to enable more targeted analyses to interrogate the biology of specific tumor subtypes, we created subtyping modules aligned with WHO molecularly-defined diagnoses.
+By working closely with pathologists and clinicians, we were able to determine a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens and determine mis-diagnosed or mis-identified samples.
```
Comment (sjspielman) [inline content/04.discussion.md:10]: I've rephrased this based on my reading that 44% of samples were EITHER identified or determined mis-identified. If my reading was incorrect, then a different rephrase would be helpful for clarity.

```suggestion
We worked closely with pathologists and clinicians to either determine a research-grade integrated diagnosis or update incorrectly diagnosed or identified samples for 44% (1260/2840) of biospecimens in the OpenPBTA cohort.
```

File: content/04.discussion.md [hunk 7]
```diff
@@ -1,3 +1,25 @@
...
+
+We applied multiple previously validated classifiers to RNA expression data.
+We predicted medulloblastoma subtypes [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263], whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076], and their degree of telomerase activity [@doi:10.1038/s41467-020-20474-9].
+By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with genomic sequencing and/or qPCR, we can quickly add knowledge to our existing biological understanding of these tumors.
+
+OpenPBTA has become a foundational layer to which we will continue to add other genomic modalities and analyses, such as germline, methylation, single cell, epigenomic, mRNA splicing, imaging, and model drug response data.
+We created the OpenPBTA project to provide robust code and data to the pediatric oncology community, as well as attract additional collaborators and new analyses to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
```
Comment (sjspielman) [inline content/04.discussion.md:17]: I think this sentence should be earlier in the discussion with in the first 3 sentences.

File: content/04.discussion.md [hunk 8]
```diff
@@ -1,3 +1,25 @@
...
+For example, RNA fusion data and the filtering module created within OpenPBTA set the stage for development of the R package, _annoFuse_ [@doi:10.1186/s12859-020-03922-7], and RShiny application, _shinyFuse_ [@url:http://shiny.imbei.uni-mainz.de:3838/shinyFuse/].
+Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed enrichment of monocyte and microglia-derived macrophages within the SHH subgroup which they suggest may accumulate following radiation therapy [@doi:10.1016/j.celrep.2021.108917].
+Expression and copy number analyses were used to demonstrate that _GPC2_ is a highly expressed and copy number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, as well as DMGs [@doi:10.1101/2021.07.06.451385].
+Foster, et. al subsequently developed a chimeric antigen receptor (CAR) directed against _GPC2_, show preclinical efficacy in mouse models.
```
Comment (sjspielman) [inline content/04.discussion.md:21]: `Foster et al.` or `Foster and colleagues`  - make consistent with Dang

**Review by sjspielman — APPROVED** [review]
I added some comments to consider throughout the text for some rephrasing, and overall looks good! I'll go ahead and approve now and you can integrate recommended changes or use your discretion to ignore them if I've gotten something wrong!

File: content/04.discussion.md (hunk 5 above)
Comment (jharenza) [reply content/04.discussion.md:8]: oh yes - generally, brain tumors are profiled by methylation in order to determine subgroupings, and we did not have that for this project (underway currently, though!)

Comment (jharenza) [reply content/04.discussion.md:8]: fyi: https://www.molecularneuropathology.org/mnp/classifiers/11

File: content/04.discussion.md [hunk 9]
```diff
@@ -1,3 +1,25 @@
...
+
+OpenPBTA has become a foundational layer to which we will continue to add other genomic modalities and analyses, such as germline, methylation, single cell, epigenomic, mRNA splicing, imaging, and model drug response data.
+We created the OpenPBTA project to provide robust code and data to the pediatric oncology community, as well as attract additional collaborators and new analyses to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
+For example, RNA fusion data and the filtering module created within OpenPBTA set the stage for development of the R package, _annoFuse_ [@doi:10.1186/s12859-020-03922-7], and RShiny application, _shinyFuse_ [@url:http://shiny.imbei.uni-mainz.de:3838/shinyFuse/].
+Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed enrichment of monocyte and microglia-derived macrophages within the SHH subgroup which they suggest may accumulate following radiation therapy [@doi:10.1016/j.celrep.2021.108917].
```
[the remark this answers is not in the document]
Comment (jharenza) [reply content/04.discussion.md:19]: ha, I was trying to be non-repetitive!

File: content/04.discussion.md [hunk 10]
```diff
@@ -1,3 +1,25 @@
...
+Expression and copy number analyses were used to demonstrate that _GPC2_ is a highly expressed and copy number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, as well as DMGs [@doi:10.1101/2021.07.06.451385].
```
[the remark this answers is not in the document]
Comment (jharenza) [reply content/04.discussion.md:20]: this one is foster as well, goes along with below but maybe I should add her name to the first one instead - will look at this again

File: content/04.discussion.md (hunk 6 above)
Comment (jharenza) [reply content/04.discussion.md:10]: does this sound better:

"We worked closely with pathologists and clinicians to determine a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as identify incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. " 

We removed mis-identified samples

File: content/04.discussion.md (hunk 7 above)
Comment (jharenza) [reply content/04.discussion.md:17]: ✅

File: content/04.discussion.md (hunk 6 above)
Comment (sjspielman) [reply content/04.discussion.md]: I like that!

**Pull request merged**